apps/extension/src/ui/domains/Mnemonic/Verify.tsx

```diff
@@ -0,0 +1,157 @@
+import { wordlist } from "@scure/bip39/wordlists/english"
+import { ChevronLeftIcon } from "@talismn/icons"
+import { classNames } from "@talismn/util"
+import { FC, useCallback, useMemo, useState } from "react"
+import { useTranslation } from "react-i18next"
+import { Button } from "talisman-ui"
+
+const WordSlot = ({
+  number,
+  active = false,
+  error = false,
+  word = "",
+}: {
+  number: number
+  active?: boolean
+  error?: boolean
+  word?: string
+}) => (
+  <span
+    className={classNames(
+      "bg-black-tertiary whitespace-nowrap rounded-xl px-6 py-3",
+      error
+        ? "text-brand-orange border-brand-orange border border-dashed"
+        : active || word
+        ? "text-body border-body border border-solid"
+        : "text-body-secondary border-body-secondary border border-dashed"
+    )}
+  >
+    <span className="select-none">{number}. </span>
+    <span className="notranslate">{word}</span>
+  </span>
+)
+
+const WordOption = ({
+  selected = false,
+  word,
+  onClick,
+}: {
+  selected?: boolean
+  word: string
+  onClick: (word: string) => void
+}) => (
+  <button
```

Comment:
missing disabled property (button looks diabled but is still clickable, i assume this is unintended)
also hover effect should be inactive if disabled, could do like this : 
`hover:bg-grey-700` => `enabled:hover:bg-grey-700`

also just FYI, classNames automatically ignores values that coerce to false, so `selected ? "text-opacity-20" : ""` could be rewritten by `selected && "text-opacity-20"`